plugins/woocommerce/tests/e2e-pw/envs/default-wpcom/playwright.config.js

```diff
@@ -8,6 +8,7 @@ config = {
 			name: 'default wpcom',
 			use: { ...devices[ 'Desktop Chrome' ] },
 			testMatch: '**basic.spec.js',
+			grepInvert: /@skip-on-default-wpcom/,
```

Comment:
Is this left intentionally? Since there are no tests out of `basic.spec.js` that we run regularly.

Reply:
I added tags for skipping tests on WPCOM. Very soon we will start adding tests also for WPCOM and this skip will be needed in the PW config. Maybe I can try now executing tests also for WPCOM since the site is available now?

I can include all the shopper tests also for WPCOM if the site is available and the tests are passing.

Reply:
Sure, but let's address WPCOM tests in a separate PR?

Reply:
Im testing now against WPCOM and it seems these shopper tests are passing there as well. I will test it once again to see if they all are passing, in which case I could include them in this PR, what do you think?

Reply:
If the change is related only to updating `playwright.config.js` for WPCOM, then I would say it's okay. Otherwise, let's address it in another PR.

Reply:
I have included all the shopper tests from this PR in the WPCOM config, [as they all seem to pass](https://d.pr/i/7rYpx6).